### ISSUE TITLE
Visual Signals: Make it possible to reorder

### ISSUE THREAD

**Issue by Zomis** [Zomis/FactorioMods#22]
Even though it is possible to hide all and then show one by one, reordering the Signal Displays would be helpful.

Add this to the detailed GUI where you can change names of them.

Reordering should ignore if it is shown or not and then when showing a hidden one it should show it in the correct order, as they appear in the renaming GUI.

**Comment by abordoli** [comment]
Yeah, the re-ordering would be great, but also would be some other minor things:
-Put the "Show X" on the left instead of the right (and even remove the word "show" leaving just the X box as I think it is self-explanatory that it is a toggle to show/not-show
-Enable features to move/expand/contract the GUI (the one where you select which ones to show)

Otherwise, I am absolutely loving this mod!
Regards,
~B

**Comment by Zomis** [comment]
I like the idea about putting "Show X" on the left and removing the label "Show".

The one about move/expand/contract the GUI is more difficult as I am not sure on how to do that. If you know of any other mod that supports that, let me know.

Thanks for liking the mod, I'm not sure how Factorio has managed without it.

**Comment by Zomis** [comment]
It's not the up and down arrows that are the problem, it's the resizing and moving of the GUI.

**Comment by abordoli** [comment]
The up and down would be fine along with the moved "X"s without the word "Show". Can that be done without worrying about the GUI resizing and movement which I feel aren't as big a deal.

**Comment by abordoli** [comment]
Just had a thought. Simply force alphabetization on the list and the user can number/letter their stuff to get it in the their preferred order...

**Comment by Zomis** [comment]
No. I think up and down arrows are much better.

**Comment by abordoli** [comment]
Sounds good. Anyway we can get a new update with the altered "show" thing? That in itself would be an awesome improvement.
